docker-compose.yml

```diff
@@ -3,6 +3,9 @@ services:
   simulator-server:
     image: registry.k8s.io/scheduler-simulator/simulator-backend:v0.2.0
     container_name: simulator-server
+    volumes:
+      - ./simulator/config.yaml:/config.yaml
+      - ./simulator/kubeconfig.yaml:/kubeconfig.yaml
```

Comment:
We shouldn't mount this kubeconfig since we don't have fake-source-cluster in this case.